.travis.yml

```diff
@@ -25,15 +25,36 @@ matrix:
       env: CHECK_CS=1
     - name: "Unit tests"
       env: PHPUNIT_CONFIG='phpunit.xml'
+      php: 7.3
+    - name: "Unit tests (PHP 7.4)"
+      env: PHPUNIT_CONFIG='phpunit.xml'
+      php: 7.4
     - name: "Admin-UI tests on Clean P"
       env:
-      - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
-      - BEHAT_OPTS="--profile=adminui --suite=adminui"
+        - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
+        - BEHAT_OPTS="--profile=adminui --suite=adminui"
+        - PHP_IMAGE=ezsystems/php:7.3-v1
+      php: 7.3
+    - name: "Admin-UI tests on Clean P (PHP 7.4)"
```

Comment:
I'm not sure all Behat jobs need to be run with both PHP 7.3 and 7.4 for every pull request. This will make the Travis queue longer and I'm not sure it provides enough value to justify that. How about something like:
We decide on a single version that most of the Behat jobs will run on (currently it's 7.3, but we can switch it to 7.4 here: https://github.com/ezsystems/ezplatform/blob/master/.env#L13-L14) and then introduce some nightly jobs that run tests on the other version to make sure we don't have any regressions?

I'd switch to 7.4 for most Behat jobs and add a couple new jobs running Behat on 7.3 during the night for open source/EE.

POV ping @ezsystems/qa-team 

The remaining non-Behat new jobs (also in https://github.com/ezsystems/ezplatform-content-forms/pull/22 and https://github.com/ezsystems/ezplatform-kernel/pull/37/files) look fine to me

Reply:
👍 for shifting some of the jobs to nightly builds for the sake of the queue.

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1333/commits/c9b92ccac1e43baa9087a38c284b146a1932a2f7